### ISSUE TITLE
`--cd` option instead specifies an output file path

### ISSUE THREAD

**Issue by pianohacker** [dpb587/gget#19]
The `--cd` option seems like it should specify an output directory, but instead seems to specify the resulting file's location:

```console
$ gget github.com/pivotal-cf/pivnet-cli --executable '*linux*' --cd ~/c/go/bin
Found 1 file (10.6M) from github.com/pivotal-cf/pivnet-cli@v2.0.2
X pivnet-linux-amd64-2.0.2 errored (processing step 2: renaming: rename /home/wjesse/c/go/.gget-679800536 /home/wjesse/c/go/bin: file exists)  
gget: error: transfers failed: pivnet-linux-amd64-2.0.2
$ gget github.com/pivotal-cf/pivnet-cli --executable '*linux*' --cd ~/c/go/bin/pivnet
Found 1 file (10.6M) from github.com/pivotal-cf/pivnet-cli@v2.0.2
√ pivnet-linux-amd64-2.0.2 done (executable)
```

This might be because `applySettings()` is called before the refs are resolved.

**Comment by dpb587** [comment]
Oh, good find. Guess it's been used with explicitly named files so far. You're right on the `applySettings` thing. Will take a look, but in the meantime you could try something like the following:

```
$ gget github.com/pivotal-cf/pivnet-cli --executable pivnet=*linux* --cd ~/c/go/bin
```

**Comment by pianohacker** [comment]
That workaround works fairly well. It also taught me about the multi-glob capabilities, which are super cool.

**Comment by dpb587** [comment]
Thanks for reporting - fixed in [v0.5.5](https://github.com/dpb587/gget/releases/tag/v0.5.5).
